packages/gateway-conformance/.aegir.js

```diff
@@ -12,50 +14,45 @@ export default {
 
       const { createKuboNode } = await import('./dist/src/fixtures/create-kubo.js')
       const KUBO_PORT = await getPort(3440)
+      const SERVER_PORT = await getPort(3441)
       const { node: controller, gatewayUrl, repoPath } = await createKuboNode(KUBO_PORT)
       await controller.start()
       const { loadKuboFixtures } = await import('./dist/src/fixtures/kubo-mgmt.js')
       const IPFS_NS_MAP = await loadKuboFixtures(repoPath)
       const kuboGateway = gatewayUrl
 
       const { startBasicServer } = await import('./dist/src/fixtures/basic-server.js')
-      const SERVER_PORT = await getPort(3441)
       const stopBasicServer = await startBasicServer({
```

Comment:
Small nit, but I initially missed that this is the VerifiedFetch based Gateway.
```suggestion
      const stopBasicServer = await startVerifiedFetchGateway({
```